packages/@okta/vuepress-site/docs/guides/pwd-optional-overview/main/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Password optional overview
```

Comment:
I thought it was called Password-optional authentication, not passwordless? The title "Password optional overview" doesn't make sense if you only call it passwordless here. If the official name is "Password optional", use that.

I'd suggest find an replace the term in the whole article and won't mention it again.